pyemma/msm/api.py

```diff
@@ -1288,6 +1292,163 @@ def bayesian_hidden_markov_model(dtrajs, nstates, lag, nsamples=100, reversible=
                                   dt_traj=dt_traj, conf=conf, store_hidden=store_hidden, show_progress=show_progress)
     return bhmsm_estimator.estimate(dtrajs)
 
+def estimate_augmented_markov_model(dtrajs, ftrajs, m, w, lag, 
+                          count_mode='sliding',  connectivity='largest',
+                          dt_traj='1 step', maxiter=500,
+                          score_method='VAMP2', score_k=10):
+    r""" Estimates an Augmented Markov model from discrete trajectories and experimental data
+
+    Returns a :class:`AugmentedMarkovModel` that
+    contains the estimated transition matrix and allows to compute a
+    large number of quantities related to Markov models.
+
+    Parameters
+    ----------
+    dtrajs : list containing ndarrays(dtype=int) or ndarray(n, dtype=int)
+        discrete trajectories, stored as integer ndarrays (arbitrary size)
+        or a single ndarray for only one trajectory.
+    ftrajs : list of trajectories of microscopic observables. Has to have
+        a one-to-one correspondence with the dtrajs. 
+    m   : Experimental averages. 
+    w   : Weight of each experimental observable. Equal to 1/(2s) where s is
+        standard error of the experimental. 
```

Comment:
If we are using Gaussians, isn't this 1/(2s^2) ?
Or how about specifying s instead? Maybe this is more intuitive.